package.json

```diff
@@ -9,8 +9,8 @@
     "@types/node": "^12.20.18",
```

Comment:
don't include package.json to the PR

Reply:
What to do?  it includes by itself.

Reply:
Add files one by one
1) git add [file-path]
2) git add [file-path]
3) git add [file-path]
4) git status (to check what can be staged)
5) git add [file-path]

Reply:
At least, that's how I do it, maybe there's a more efficient way, but whatever

Reply:
And you can also use Tab for autocompletion

Reply:
I also add them manually ...maybe missed this one